keymanapp/keyman
# chore(windows): removed cached context windows engine

Fixes #10052 

# User Testing

I have taken the context and store test for the acceptance tests on the wiki. We don't need the stores will just use the 
 **043 - output and keystroke.kmx** and **045 - deadkey and context.kmx**
Use the text editor available on this PR in the User Testing section

- GROUP_TEXT_EDITOR
- GROUP_WORD_PAD  (TSF)


#### Test cases

<details>
<summary>click to expand</summary>

The key referred to as "enter" is alternatively labelled "return" on some keyboards.

Keyboards for the following test are found in the zip file [store_context_kbd.zip](https://github.com/keymanapp/keyman/files/9441852/store_context_kbd.zip)

**TEST_OUTPUT_KEYSTROKE**
1. Install **043 - output and keystroke.kmx**
2. Open `Text Editor`
3. Select  043 - output and keystroke.kmx as the current keyboard.
4. Type <kbd>1</kbd><kbd>2</kbd><kbd>3</kbd>

Expected Result: "abd3"

**TEST_OUTPUT_KEYSTROKE_INVALID_CONTEXT**

1. If not installed from the previous test - Install **043 - output and keystroke.kmx**
2. Open `Text Editor`
3. Select  043 - output and keystroke.kmx as the current keyboard.
4. Type <kbd>1</kbd><kbd>2</kbd><kbd>Enter</kbd><kbd>3</kbd>

Expected Result: \
"abc\
3"

**TEST_OUTPUT_KEYSTROKE_CONTEXT_FRAME_KEY**

1. If not installed from the previous test - Install **043 - output and keystroke.kmx**
2. Open `Text Editor`
3. Select  043 - output and keystroke.kmx as the current keyboard.
4. Type <kbd>1</kbd><kbd>2</kbd><kbd>left arrow key</kbd><kbd>right arrow key</kbd><kbd>3</kbd>

Expected Result WordPad: "abd3"

Expected Result TextEditor: "abc3"


**TEST_DEADKEY_AND_CONTEXT**
1. Install **045 - deadkey and context.kmx**
2. Open `Text Editor`
3. Select  045 - deadkey and context.kmx as the current keyboard.
4. Type <kbd>y</kbd><kbd>z</kbd><kbd>Shift</kbd>+<kbd>/</kbd>

Expected Result: "correct"

**TEST_DEAD_CONTEXT_FRAME_KEY**
1. Install **045 - deadkey and context.kmx**
2. Open `Text Editor`
3. Select  045 - deadkey and context.kmx as the current keyboard.
4. Type <kbd>y</kbd><kbd>z</kbd> <kbd>left arrow key</kbd><kbd>right arrow key</kbd><kbd>Shift</kbd>+<kbd>/</kbd>

Expected Result: "\<ab\>?"
</details>
